java/src/client/game/GameManager.java

```diff
@@ -25,20 +26,41 @@
     /**
      * The server that may be a fake one or a real one
      */
-    IServer mServer;
+    private IServer mServer;
+    /**
+     * Handel to a facade for the current game, allows for use on both client and server - with n games.
```

Comment:
I scrolled through quickly and I have a _very_ sensitive eye. With the new review system it's click, type, control enter. The time taken to write both of these comments has taken more time.